manifests/init.pp

```diff
@@ -375,9 +391,7 @@
       group   => $step_tickers_group,
       mode    => $step_tickers_mode,
       content => template('ntp/step-tickers.erb'),
-      require => [ Package[$package_name_real],
```

Comment:
sorry for not understanding completely, what do you mean by saying "before would have to be conditional based on package_manage " ?  or is there anything i need to change?